mappings/net/minecraft/client/color/world/BiomeColors.mapping

```diff
@@ -0,0 +1,28 @@
+CLASS dka net/minecraft/client/color/world/BiomeColors
+	CLASS dka$a Provider
+		METHOD getColor (Lbie;Lev;)I
+			ARG 1 biome
+			ARG 2 pos
+	FIELD a GRASS_COLOR Ldka$a;
+	FIELD b FOLIAGE_COLOR Ldka$a;
+	FIELD c WATER_COLOR Ldka$a;
+	FIELD d WATER_FOG_COLOR Ldka$a;
+	METHOD a getGrassColor (Lbgp;Lev;)I
```

Comment:
What are thoughts on `At` suffix for these?

Reply:
The suffix would be redundant, and conflict with Yarn's (and Java's) current naming conventions. An `At` suffix would be something for another PR if it is something you are interested in - it is out of scope for this PR, and changes would need to be reflected across the entire codebase with methods such as `getBlockState`/`getBiome`/`getBlockEntity` for naming consistency.

Reply:
Essentially, the positional behaviour of the method is already implied by the `BlockPos` parameter, much like an indexing behaviour is implied through the use of `List#get(int)`

Reply:
Ah, okay. That does make sense. Thanks